### ISSUE TITLE
undefined: sqlite3.Error issue

### ISSUE THREAD

**Issue by abdyek** [livebud/bud#405]
When create a hello project, the CLI trows this error to me
```sh
⬢[abdyek@toolbox Go]$ bud create hello
| Created: go.mod
| Created: package.json
| Created: .gitignore
| Created: public/favicon.ico
| Installed: go modules
| Installed: node modules
# github.com/livebud/bud/internal/dag
../../../../go/pkg/mod/github.com/livebud/bud@v0.2.8/internal/dag/sqlite.go:345:27: undefined: sqlite3.Error
../../../../go/pkg/mod/github.com/livebud/bud@v0.2.8/internal/dag/sqlite.go:349:47: undefined: sqlite3.ErrReadonlyDbMoved
| exit status 1
```

**Comment by DianWoung** [comment]
same question

**Comment by matthewmueller** [comment]
Hmm, that's odd. What OS are you running on? Did you try running `go mod tidy`?

I sense it's related to: https://github.com/mattn/go-sqlite3/issues/384

**Comment by DianWoung** [comment]
this err shows on windows，I use intel Mac is ok

**Comment by ErSauravAdhikari** [comment]
I am still experiencing this issue.
![image](https://github.com/livebud/bud/assets/69170305/d3fcd602-cf27-4dba-9abd-821f1ea13a0f)

I tried to use `go mod tidy` as @matthewmueller pointed out. But this did not work as well.

Env: Linux/Mint

**Comment by ErSauravAdhikari** [comment]
I've manged to identify the problem. 
This is related to CGO.

For more information you can check about static builds on google. 

TLDR Fix: 
```
saurav@saurav:~/Workspace/livebud/gandaki$ go env -w CGO_ENABLED=1
```

I think we should look more into this. As ideally it would be better if works irrespective of CGO_ENABLED=1/0.
